File: FitTrackApp/app/src/main/java/com/example/fittrackapp/models/Exercise.java
package com.example.fittrackapp.models;

public class Exercise {
    public int exerciseId;
    public String exerciseName;
    public String muscleGroup;
    public String aliasName;
    public String qrCode;

    public Exercise(String exerciseName, String muscleGroup, String aliasName, String qrCode) {
        this.exerciseName = exerciseName;
        this.muscleGroup = muscleGroup;
        this.aliasName = aliasName;
        this.qrCode = qrCode;
    }
}

File: backend/models/Exercise.cs
using System.ComponentModel.DataAnnotations;

namespace FitTrackBackend.Models;

public class Exercise
{
    [Key]
    public int ExerciseId { get; set; }

    public string ExerciseName { get; set; } = "";

    public string MuscleGroup { get; set; } = "";

    public string AliasName { get; set; } = "";

    public string QrCode { get; set; } = "";
    
    public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
}